Number_types/include/CGAL/NT_wrapper.h

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2024, GeometryFactory Sarl (France). All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Laurent Rineau
+
+#ifndef CGAL_NT_WRAPPER_H
+#define CGAL_NT_WRAPPER_H
+
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Exact_predicates_exact_constructions_kernel.h>
+#include <CGAL/Coercion_traits.h>
+#include <CGAL/Real_embeddable_traits.h>
+#include <CGAL/Kernel/global_functions.h>
+#include <CGAL/Kernel/Type_equality_wrapper.h>
+#include <CGAL/functional.h>
+#include <functional>
+#include <source_location>
+
+namespace CGAL {
+
+template <typename NT>
+class NT_wrapper {
```

Comment:
In case it is not documented it should go into `CGAL/Number_types/internal`.
And a minimal documentation in a comment what the class does is not a mistake.

Reply:
I wonder if it should be documented. What do you think?

Reply:
Write already just a short C++ comment so that we get an idea what it is for.  Who do you expect to use it. Just you as you needed it for debugging?  Or CGAL developers in general?  Or even users?